helpers/constants.js

```diff
@@ -110,7 +110,6 @@ var constants = {
 		offset: 1451520, // Start rewards at block (n)
 		distance: 3000000, // Distance between each milestone
 	},
-	// WARNING: When changing totalAmount you also need to change getBlockRewards(int) SQL function!
```

Comment:
Why removed this comment? Is is irrelevant now?

Reply:
Yep, that SQL function was removed.